common/lib/xmodule/xmodule/course_module.py

```diff
@@ -846,6 +846,15 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    teams_configuration = Dict(
+        display_name=_("Teams Configuration"),
+        help=_(
+            "Enter configuration for the teams feature. Expects two entries: max_team_size and topics, where"
```

Comment:
You'll want a space after "where" i.e. `"... two entries: max_team_size and topics, where " so that we don't concatenate "where" and "topics" into "wheretopics"